src/components/modals/Modal.vue

```diff
@@ -3,11 +3,17 @@
     <div class="modal-backdrop" @click="close">
       <div
         class="modal"
+        v-bind:class="[isMaximized ? 'maximized' : 'not-maximized']"
         role="dialog"
         aria-labelledby="modalTitle"
         aria-describedby="modalDescription"
         @click.stop=""
       >
+        <font-awesome-icon
+          @click="toggleMaximized"
+          class="maximize-toggle"
+          icon="window-maximize"
```

Comment:
The icon should switch between maximize and minimize:
```suggestion
          :icon="isMaximized ? 'window-minimize' : 'window-maximize'"
```

Reply:
will add

Reply:
unfortunately it doesn't evaluate the conditional in these circumstances >_>